app/Http/Controllers/MinesController.php

```
@@ -0,0 +1,273 @@
+
+        $user = auth('sanctum')->user();
+
+        if (!$freeBet && !$user) {
+            return response()->json(['message' => 'You should be authenticated to do that'], 401);
+        }
+
+        if (!$freeBet && $user->coins < $data['amount']) {
+            return response()->json(['message' => 'Insufficient balance'], 400);
+        }
+
+        $balance = $user ? $user->coins : 0;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$balance'. (Unused Code Rules)

(UnusedLocalVariable)